spec/models/absolute_path_validator_spec.rb

```diff
@@ -1,11 +1,11 @@
 require "rails_helper"
 
-describe AbsolutePathValidator, type: :model do
-  ValidatablePath = Struct.new(:path) do
-    include ActiveModel::Validations
-    validates :path, absolute_path: true
-  end
+ValidatablePath = Struct.new(:path) do
```

Comment:
This would be good to move into the test. I think it could be 

```
let(:validatable_path_class) do
  Struct ...
end
```

Similar to: https://github.com/alphagov/email-alert-api/blob/6f733b3f828405b8b46dfe9c3f673cd177593c70/spec/validators/tags_validator_spec.rb#L2-L11